mirror-plugins

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env ruby
+
+require "rubygems"
+require "bundler/setup"
+require "dotenv"
+require "gitlab"
+
+Dotenv.load
+
+Gitlab.configure do |config|
+  config.endpoint = ENV["GITLAB_API_ENDPOINT"]
```

Comment:
I strongly recommend using `#fetch` to get an entry from a Hash. If the key doesn't exist `#[]` will return `nil` and `#fetch` will raise an error. In this situation, a `nil` will break unexpectedly at some point later, potentially causing more trouble, and definitely making it harder to track down the source of the issue.

Reply:
Thanks, I didn't know that difference in behaviour. I've switched all the `ENV` calls to `ENV.fetch('VAR_NAME')` now.